charts/console/templates/deployment.yaml

```
@@ -60,7 +60,7 @@ spec:
         {{- end }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ include "console.imageRegistry" . }}/{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: {{ include "console.container.image" . | quote }}
```

nit: it can only be a string. quotes are unneeded. not a blocker.
```suggestion
          image: {{ include "console.container.image" . }}
```